README.md

```diff
@@ -50,29 +50,454 @@ resolvers += "jitpack" at "https://jitpack.io"
+To configure hot reloading with webpack devserver, check out [build.sbt](https://github.com/OutWatch/seed.g8/blob/master/src/main/g8/build.sbt) and [webpack.config.dev.js](https://github.com/OutWatch/seed.g8/blob/master/src/main/g8/webpack.config.dev.js) from the [g8 template](https://github.com/OutWatch/seed.g8).
 
 
 ## Bugs and Feedback
+For bugs, questions and discussions please use [Github Issues](https://github.com/OutWatch/outwatch/issues).
 
-For bugs, questions and discussions please use the [Github Issues](https://github.com/OutWatch/outwatch/issues).
 
 ## Community
+We adopted the [Typelevel Code of Conduct](http://typelevel.org/conduct.html). People are expected to follow it when discussing OutWatch on the Github page, Gitter channel, or other venues.
+
+
+## Documentation and Examples
+Outwatch is a web frontend UI framework written in ScalaJS.
+
+If you find any error in the examples, please open an issue on GitHub.
+
+There is a changelog which contains examples of the latest changes: [CHANGELOG.md](CHANGELOG.md)
+
+There is also the outdated but conceptually still correct [documentation](https://outwatch.github.io/) -  [contributions welcome](https://github.com/OutWatch/outwatch.github.io).
+
+### Hello World
+In your html file, create an element, which you want to replace by dynamic content:
+
+```html
+...
+<body>
+    <div id="app"></div>
+    <!-- your compiled javascript should be imported here -->
+</body>
+...
+```
+
+To render html content with outwatch, create a component and render it into the given element:
+
+```scala
+import outwatch.dom._
+import outwatch.dom.dsl._
+
+object Main {
+  def main(args: Array[String]): Unit = {
+    
+    val myComponent = div("Hello World")
+
+    OutWatch.renderReplace("#app", myComponent).unsafeRunSync()
+  }
+}
+```
+
+Running `Main` will replace `<div id="app"></div>` with the content defined in `myComponent`:
+
+```html
+...
+<body>
+    <div>Hello World</div>
+    ...
+</body>
+...
+```
+
+### Static Content
+First, we will focus on creating immutable/static content that will not change over time. The following examples illustrate to construct and transform HTML/SVG tags, attributes and inline stylesheets.
+
+#### Concatenating Strings
+```scala
+div("Hello ", "World")
+// <div>Hello World</div>
+```
+
+
+#### Nesting
+```scala
+div(span("Hey ", b("you"), "!"))
+// <div>Hey <b>you</b>!</div>
+```
+
+
+#### Primitives
+```scala
+div(true, 0, 1000L, 3.0)
+// <div>true010003.0</div>
+```
+
+
+#### Attributes
+Attributes are put inside the tag.
+
+```scala
+div(id := "test")
+// <div id="test"></div>
+```
+
+The order of content and attributes does not matter.
+
+```scala
+div("How ", id := "test", "are", title := "cool", " you?")
+// <div id="test" title="cool">How are you?</div>
+```
+
+
+#### Styles
+Styles are also written into the tag. All style properties have to be written in *camelCase*.
+
+```scala
+div(color := "tomato", "Hello")
```

Comment:
```suggestion
div(color := "blue", "Hello")
```

Reply:
why not tomato? :grin:

Reply:
Is there a CSS color named 🍅? 😁

Reply:
yes! It's a nice red color :ok_hand:: https://www.quackit.com/html/codes/color/color-tester.cfm?colorname=tomato